superset-frontend/webpack.config.js

```diff
@@ -60,14 +60,39 @@ if (isDevMode) {
 
 const plugins = [
   // creates a manifest.json mapping of name to hashed output used in template files
-  new WebpackAssetsManifest({
-    publicPath: true,
+  new ManifestPlugin({
+    publicPath: output.publicPath,
+    seed: { app: 'superset' },
     // This enables us to include all relevant files for an entry
-    entrypoints: true,
+    generate: (seed, files, entrypoints) => {
+      // Each entrypoint's chunk files in the format of
+      // {
+      //   entry: {
+      //     css: [],
+      //     js: []
+      //   }
+      // }
+      const entryFiles = {};
+      for (const [entry, chunks] of Object.entries(entrypoints)) {
+        entryFiles[entry] = {
+          css: chunks
+            .filter(x => x.endsWith('.css'))
+            .map(x => path.join(output.publicPath, x)),
+          js: chunks
+            .filter(x => x.endsWith('.js'))
+            .map(x => path.join(output.publicPath, x)),
+        };
+      }
+      return {
+        ...seed,
+        // files: filePaths,
+        entrypoints: entryFiles,
+      };
```

Comment:
Note that the new `manifest.json` will now not emit file entries at the root, e.g.:

```json
{
    "filename.js": "filename.[hash].js"
}
```

It will not only contain an `entrypoints` section (the only thing that's been used by both Flask and `webpack-dev-server`), a list of CSS/JS chunk files for each bundle.